modules/ocf/files/packages/cups/raster-filter

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# A Tea4CUPS filter to perform rasterization.
+# Based off the pdf-open script
+
+set -euo pipefail
+
+input=$(mktemp --suffix='.pdf')
+
+# Feed filter input into temporary file
+cat - > "$input"
+
+# If the file is not a PDF, then just pass it thorugh
+# (Note that all PDFs start with "%PDF")
+if [ "$(head -c 4 "$input")" != "%PDF" ]; then
+    cat "$input"
+    exit
+fi
+
+# Rasterizing PDF...
+output=$(mktemp --suffix='.pdf')
+
+if ! convert -density 250 "$input" "$output" 2> /tmp/error; then
+    echo 'Convert failed.'
```

Comment:
Have you ever seen a conversion fail? If this happens, will it get reported to us (root@ mailing list) and will we be able to see the error message?

Reply:
With previous versions of the setup, I have seen some conversions fail. (That's one of the reasons I've decided to up the ImageMagick memory limit.) So far, the script doesn't really do anything other than output stderr to `/tmp/error`, so I'm going to try to change that so an email gets sent out.

Reply:
I'd recommend writing a script in `utils` that does something similar to https://github.com/ocf/ldapsync/blob/master/ldapsync/email_ldapsync_problem

Where it runs a command (or takes stdin) and sends a failure email to ocfroot if anything gets output.

Reply:
Sounds good, I'm basing my script off the example you gave and parts of enforcer. 

The OCF utils repo mentions that util scripts should be for scripts "which might be executed by a person." Since the script I'm writing preferably should only be executed by my filter, I'm going to look into placing it under `/usr/local/bin`. (The enforcer script is also placed at the same location, but on whiteout.) Or is this not a good location?